website/client/components/creatorIntro.vue

```diff
@@ -905,7 +908,7 @@ export default {
       let tasks = response.data.data;
       tasks.forEach(task => {
         this.$store.state.user.data.tasksOrder[`${task.type}s`].unshift(task._id);
-        Object.assign(this.$store.state.tasks.data[`${task.type}s`][0], task);
+        this.$store.state.tasks.data[`${task.type}s`].unshift(task);
```

Comment:
can we use the tasks create action here? the code is the same

Reply:
It is a little different, but I can modify the store.